test/awss3.jl

```diff
@@ -182,12 +181,7 @@ end
 end
 
 @testset "Empty and Delete Bucket" begin
-    for v in s3_list_versions(aws, bucket_name)
-        s3_delete(aws, bucket_name, v["Key"]; version = v["VersionId"])
-    end
-
-    s3_delete_bucket(aws, bucket_name)
-    @test !in(bucket_name, s3_list_buckets(aws))
```

Comment:
Should you add this test back? Or was it testing wrong?

Reply:
This test is really just fluff, because the next test underneath this attempts to delete an already deleted bucket.